content/_index.md

```diff
@@ -8,6 +8,41 @@ cascade:
     showViews : false
 ---
 
-We are thrilled to have you join our community of innovators and creators in the exciting world of Espressif products! Whether you're a seasoned developer or just starting your journey, this portal is your go-to destination for a treasure trove of information, tutorials, and resources centered around our cutting-edge technologies.
+"Welcome to the [Espressif Systems](https://espressif.com/) Developer Portal—your official hub for all things ESP32 and more. Explore our extensive collection of articles, workshops, and tutorials to enhance your development journey with the latest tools and insights."
 
-Join us on this exciting journey of exploration, innovation, and collaboration. The Espressif Developer Portal is not just a platform; it's a community-driven space where ideas flourish and projects come to life.
+## Featured Article
+
+{{< article link="/blog/meet-espressif-developer-portal/" >}}
+
+<br>
+{{< button href="#button" target="_self" href=/blog/ >}}
+More Articles
+{{< /button >}}
+
+## Workshop Highlight
+
+Dive into our latest workshops and master the skills you need to maximize the power of the ESP32.
+
+{{< article link="/workshops/iot-introduction-with-rainmaker/" >}}
+
+<br>
+{{< button href="#button" target="_self" href=/workshops/ >}}
+More Workshops
+{{< /button >}}
+
+## New Chip Support Status for ESP-IDF
```

Comment:
We moved this support status page from ESP-IDF's GitHub because the status also mentions other projects, not just ESP-IDF. SUggest:

```suggestion
## New Chip Support Status
```